[Menu] Include 'list' in class key

rbrishabh opened the pull request:
Fixes #17202 

Hey @oliviertassinari !

Let me know if this needs any changes! Thanks! =)

rbrishabh commented:
Ahh! Okay. I'll make sure to move on to more challenging issues.
If you have any particular issue in mind, please tag me and I'd be more than happy to look at it! @oliviertassinari

=)

oliviertassinari commented:
@rbrishabh We have a couple of issues that are worth exploring. We try to add the label to "important" for issues that can significantly impact the community. We also have this concept of "help appreciated" for issues that have, a more or less clear resolution path. If you are using Material-UI, and have faced problems in the past, addressing the concerns, is also a great way to contribute!

eps1lon commented:
@rbrishabh Much appreciated, thanks!

rbrishabh commented:
> @rbrishabh We have a couple of issues that are worth exploring. We try to add the label to "important" for issues that can significantly impact the community. We also have this concept of "help appreciated" for issues that have, a more or less clear resolution path. If you are using Material-UI, and have faced problems in the past, addressing the concerns, is also a great way to contribute!

Alright! On to some more complex (or less trivial issues now)! As always, thanks for the guidance! @oliviertassinari @eps1lon
